src/Core/src/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -2206,7 +2206,6 @@ override Microsoft.Maui.MauiAppCompatActivity.OnActivityResult(int requestCode,
 override Microsoft.Maui.MauiAppCompatActivity.OnBackPressed() -> void
 override Microsoft.Maui.MauiAppCompatActivity.OnConfigurationChanged(Android.Content.Res.Configuration! newConfig) -> void
 override Microsoft.Maui.MauiAppCompatActivity.OnCreate(Android.OS.Bundle? savedInstanceState) -> void
-override Microsoft.Maui.MauiAppCompatActivity.OnDestroy() -> void
```

Comment:
@mattleibow shouldn't this be marked as remove on Unshipped ?

Reply:
It probably should have, but it is an override so we can let it slide...